Crop - keystone correction: correction guide hard to adjust

On Ubuntu 20.04 with KDE, self-compiled cd8e1df2.

When **'reduce resolution of preview image'** (**preview_downsampling**) is set to **1/4**, I find the guides for adjusting keystone in the crop module very hard to adjust; there is **no problem when the setting is left at 'original'**.
I have to very accurately target the control point until the cursor switches into cross-hairs (from the 4-way arrows). I do not recall this behaviour with previous versions.

ok, did that and rebuilt

but did not work and I did check that I pasted the 3 lines into the correct
location.

3160,3162d3159
< const float pr_d = darktable.develop->preview_downsampling;
< // slightly adjust the size of keystone control area depending on the downsampling
< const float ext = DT_PIXEL_APPLY_DPI(0.005f + ((1.0f - pr_d) / 100.0f)) / zoom_scale;

[ 84%] Built target toneequal
gmake: *** [Makefile:171: all] Error 2

still workking at it.

--- a/src/iop/clipping.c
+++ b/src/iop/clipping.c
@@ -3153,7 +3153,9 @@ int mouse_moved(struct dt_iop_module_t *self, double x, double y, double pressur
     dt_control_change_cursor(GDK_FLEUR);
     g->straightening = g->cropping = 0;
     // or maybe keystone
-    const float ext = DT_PIXEL_APPLY_DPI(0.005f) / zoom_scale;
+    const float pr_d = darktable.develop->preview_downsampling;
+    // slightly adjust the size of keystone control area depending on the downsampling
+    const float ext = DT_PIXEL_APPLY_DPI(0.005f + ((1.0f - pr_d) / 100.0f)) / zoom_scale;
     if(g->k_show == 1 && g->k_drag == FALSE)
     {
       float pts[2] = { pzx * wd, pzy * ht };

